src/string_bytes.cc

```diff
@@ -182,15 +185,15 @@ static const int8_t unhex_table[256] =
     -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1
   };
 
-#define unhex(x)                                                              \
-  static_cast<unsigned>(unhex_table[static_cast<uint8_t>(x)])
-
+static inline unsigned unhex(uint8_t x) {
+  return unhex_table[x];
+}
```

Comment:
You’re right, this is just because it seems a lot cleaner to me and I stumbled across it while doing this.